src/extension/app/store/app.js

```diff
@@ -688,26 +688,20 @@ export class AppStore {
         detail: { data },
       }));
       const userEvents = [
-        'shown',
```

Comment:
Removed unused events that carried over from the old extension.

Reply:
I think this would be a good time to consolidate to a single list of event names in `EXTERNAL_EVENTS` instead of duplicating them here.